src/client/components/Meal.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
+const Meal = () => {
```

Comment:
This is all nice and clean and clear! 
However, I have some roubles 🤔 
- A meal component should display the meal. Why is it fetching the meal and not taking it as prop? 
- The meal component also has all the forms to submit a review or a reservation - that is unexpected! Should it be 2 more separate components that are just used here, connected together by the meal id from props? 💡

Reply:
This is a great idea! i didn’t think about separating since just wanted to finish before deadline and i thought in the meal page it would be easy creating reservation or review. Other wise i thought if i put these form in the home page i need to fetch meals maybe as a select list but i didn’t have that time :) but definitely i'll improve and play more about this project 🤞🏼Thank you for reviewing my hw 🫶🏼